src/Extension/ImageClickSpotsExtension.php

```diff
@@ -15,12 +17,11 @@ class ImageClickSpotsExtension extends DataExtension
     ];
 
     private static $has_many = [
-        'DataPoints' => DataPoints::class.'.Parent',
+        'DataPoints' => DataPoint::class.'.Parent',
     ];
 
-    public function getCMSFields()
+    public function updateCMSFields(FieldList $fields)
```

Comment:
This is an extension so I had to update the method to work as an extension method